language/diem-framework/modules/0L/FullnodeSubsidy.move

```diff
@@ -17,7 +17,8 @@ address 0x1 {
     use 0x1::TowerState;
 
     public fun get_proof_price(one_val_subsidy: u64): u64 {
-      let global_proofs = TowerState::get_fullnode_proofs();
+
+      let global_proofs = TowerState::get_fullnode_proofs_in_epoch_above_thresh();
```

Comment:
This function needs divide by 0 protection. Simply need to return 0 if global_proofs is 0

Reply:
done